languages.yaml

```diff
@@ -17,7 +16,6 @@ Abap:
       - "#EEEEEE"
     chip: "#E8274B"
```

Comment:
Maybe the chip `color` could also be given by `gengo` ?

Reply:
Actually, if that's the only reason to keep it, we might as well remove it 🤔

Reply:
Yeah, that sounds good to me. I didn't add it initially since gengo's colors are different (onefetch's are all from linguist, right?)

Guess we could still get gengo's colors into the vercel UI with WASM, but that feels like major overkill :laughing:

Reply:
This is just how I prefer how PRs/diffs are made, but I'm thinking we could remove chip colors and use gengo's in a separate PR.